app/controllers/errors_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ErrorsController < BareApplicationController
```

Comment:
Given that this is now mainly for authenticated users, I think we should review the use of the BareApplicationController here as well as the external layout.

Reply:
Great point